app/assets/javascripts/views/story_view.js

```diff
@@ -541,34 +542,29 @@ const StoryView = FormView.extend({
   },
 
   renderCollapsed: function(isGuest) {
-
     this.$el.removeClass('editing');
     this.$el.html(this.template({story: this.model, view: this}));
-    this.$el.toggleClass('collapsed-iteration',
-                         !this.model.get('isVisible') &&
-                         !this.isSearchResult);
+    this.$el.toggleClass('collapsed-iteration', !this.model.get('isVisible') && !this.isSearchResult);
 
     const stateButtons = this.$('[data-story-state-buttons]').get(0)
     if(stateButtons) {
-      ReactDOM.render(
-        <StoryStateButtons
-          events={this.model.events()}
-        />,
-        stateButtons
-      );
+      ReactDOM.render(<StoryStateButtons events={this.model.events()} />, stateButtons);
     }
 
     const estimateButtons = this.$('[data-story-estimate-buttons]').get(0)
     if(estimateButtons) {
       ReactDOM.render(
-        <StoryEstimateButtons
-          points={this.model.point_values()}
-          onClick={this.estimate}
-        />,
+        <StoryEstimateButtons points={this.model.point_values()} onClick={this.estimate} />,
         estimateButtons
       );
     }
 
+    const copyStoryIdClipboardLink = this.$('[data-story-id-copy-clipboard]').get(0)
+    if(copyStoryIdClipboardLink) {
+      ReactDOM.render(<StoryCopyIdClipboard id={this.id} />, copyStoryIdClipboardLink)
+      new Clipboard('.story-id');
```

Comment:
Can we use it like this instead? https://github.com/Codeminer42/cm42-central/blob/d83c594c78612c9161f9c007844141d7aab7f936/app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryHistoryLocation.js#L30-L37

Reply:
I will take a look